cargo-process.el

```diff
@@ -336,6 +349,24 @@ Without the prefix argument, return DEFAULT immediately."
       (read-shell-command "Cargo command: " default)
     default))
 
+(defun cargo-process--get-dependencies (manifest)
+  "Extract the list of dependencies from the
+MANIFEST (i.e. Cargo.toml)."
+  (save-excursion
+	(let ((buffer (find-file-noselect manifest))
+		  (deps-list))
+	  (set-buffer buffer)
+	  (setq buffer-read-only t)
```

Comment:
Why are we making `Cargo.toml` read-only ? After the command is finished / canceled it will stay read only.